README.md

```diff
@@ -34,12 +34,22 @@ The operator will then create a CatalogSourceConfig CR which will, for the time
 
 #### In an OKD Cluster
 ```bash
-$ oc apply -f deploy/
+$ oc apply -f deploy/catalogsourceconfig.crd.yaml
+$ oc apply -f deploy/operatorsource.crd.yaml
+$ oc apply -f deploy/service_account.yaml
+$ oc apply -f deploy/role.yaml
+$ oc apply -f deploy/role_binding.yaml
+$ oc apply -f deploy/operator.yaml
 ```
 
 #### In a Kubernetes Cluster
```

Comment:
Do we need to repeat this for the k8s? Can't we have one section for k8s saying something to the effect of "execute the above commands in the same order using `kubectl`?

Reply:
I have updated the README as requested.